osu.Game.Tournament/Screens/Setup/SetupScreen.cs

```diff
@@ -19,7 +19,7 @@
 
 namespace osu.Game.Tournament.Screens.Setup
 {
-    public class SetupScreen : TournamentScreen, IProvideVideo
+    public class SetupScreen : TournamentScreen
```

Comment:
Not sure how big of a deal this is, but the setup screen was (in a dodgy way) forcing the main video to hide by inheriting `IProvideVideo` despite not having any actual video background sprite.

With this PR, now the main video shows on setup screen.

Reply:
Well okay, this is probably a bit bad. Disabling automerge for now.

Reply:
I knew this was the case but didn't really think too much of it. I've added a box to hide it.